src/Paystack.php

```diff
@@ -272,9 +272,9 @@ public function isTransactionVerificationValid($transaction_id = null)
      * @return json
      * @throws PaymentVerificationFailedException
      */
-    public function getPaymentData()
+    public function getPaymentData($reference)
```

Comment:
Pls assign `$reference` a default value of `null`

Reply:
So the workaround i went with was extending the class like this 
![image](https://github.com/unicodeveloper/laravel-paystack/assets/25669630/509fa0b5-06fc-4492-9b7a-55327f865191)


and handled this way 
![image](https://github.com/unicodeveloper/laravel-paystack/assets/25669630/8299c7cc-59c4-4b1f-bd1f-19f57566e8c2)

Reply:
This was without making any changes with your initial code. it works just fine